lms/djangoapps/teams/tests/test_views.py

```diff
@@ -166,6 +168,20 @@ def setUp(self):
         self.test_team_3.add_user(self.users['student_enrolled_both_courses_other_team'])
         self.test_team_5.add_user(self.users['student_enrolled_both_courses_other_team'])
 
+    def create_and_enroll_student(self, course=None):
+        """ Creates a new student and enrolls that student in the course.
+
+        Adds the new user to the self.users dictionary with the username as the key.
+
+        Returns the username once the user has been created.
+        """
+        user = UserFactory.create(password=self.test_password)
```

Comment:
It would be nice to use this new method in setUp to simplify lines 132-143 and 155-165 (except for student_unenrolled).